docs/core/versions/binding.md

```diff
@@ -0,0 +1,170 @@
+- For a set of projects, placed at a common [grand-]parent for all of the projects. This location can be as high in the directory structure as the root of a drive.
+
+The following example shows the `global.json` syntax:
+
+``` json
+{
+  "sdk": {
+    "version": "1.0.0"
+  }
+}
+```
+
+You only use `global.json` to specify a version when you must lock a specific app or set of apps to an earlier SDK than the latest on the machine.
+
+The process for selecting an SDK version is:
+
+- `dotnet` binds to latest SDK by default.
+- You can override the latest version using global.json.
+- `dotnet` uses the first global.json file found, iteratively reverse-navigating the path upward from the current working directory.
+
+## Select a target SDK
+
+You specify the target framework in the project file. Set the `TargetFramework` as shown in the following example:
+
+``` xml
+<TargetFramework>netcoreapp2.0</TargetFramework>
+```
+
+You set multiple target frameworks if you need to build different code for different environments. Setting multiple target frameworks is more common for libraries, but can be done with applications as well. You specify a `TargetFrameworks` property (plural of `TargetFramework`). The target frameworks will be semicolon-delimited as shown in the following example:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+```
+
+A given SDK will support a fixed set of frameworks, typically capped to the target framework of the runtime(s) it includes. For example, the .NET Core 2.0 SDK includes the .NET Core 2.0 runtime, which is an implementation of the `netcoreapp2.0` target framework. The .NET Core 2.0 SDK will support `netcoreapp1.0`, `netcoreapp1.1`, and `netcoreapp2.0` but not `netcoreapp2.1` (or higher). You'll need to install the .NET Core 2.1 SDK in order to build for `netcoreapp2.1`.
+
+.NET Standard target frameworks are also capped in the same way. The .NET Core 2.0 SDK will be capped to `netstandard2.0`.
+
+A given SDK defines a fixed minimum runtime patch version for each target framework that it supports. The minimum runtime patch version is maintained at the `.0` version (for example, `2.0.0`) for the lifetime of a major/minor runtime version family.
+
+You can override the minimum runtime patch version (to higher or lower versions) in the project file, as shown in the following example:
+
+``` xml
+<RuntimeFrameworkVersion>2.0.4</RuntimeFrameworkVersion>
+```
+
+When you target multiple target frameworks, you must specify the minimum runtime patch version to the specific .NET Core target framework, as shown in the following example:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+<RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.4</RuntimeFrameworkVersion>
+```
+
+A given SDK supports a fixed set of target frameworks. Typically, the target frameworks are the SDK framework version and a set of earlier versions.
```

Comment:
Two things...

There seems an implication here that the frameworks are only .NET Core frameworks. 

@livarcocc don't we allow targeting all previous .NET Core frameworks?

Reply:
I'll assume @KathleenDollard is correct that all previous .NET Core faremworks are allowed. @livarcocc please correct if necessary.